### PR TITLE
feat(compiler): List exports in wasm_utils

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

File: compiler/src/utils/wasm_utils.re [hunk 1]
```diff
@@ -279,6 +286,27 @@ let get_wasm_sections = (~reset=false, inchan) => {
             );
           let true_offset = pos_in(inchan);
           (Custom(name), true_offset, size - (true_offset - offset));
+        | Export(_) =>
+          let num_exports = Int32.to_int(read_leb128_u32_input(inchan));
+          let rec read_export = () => {
+            let name_len = Int32.to_int(read_leb128_u32_input(inchan));
+            let name = really_input_string(inchan, name_len);
```
Comment (peblair) [inline compiler/src/utils/wasm_utils.re:293]: gets me every time

**Review by peblair — APPROVED** [review]
:tada:

File: compiler/src/utils/wasm_utils.re (hunk 1 above)
Comment (ospencer) [reply compiler/src/utils/wasm_utils.re:293]: but _really_

**Review by phated — APPROVED** [review]
🔥 🔥 🔥 🔥  Merge that!

File: compiler/src/utils/wasm_utils.re (hunk 1 above)
Comment (phated) [reply compiler/src/utils/wasm_utils.re:293]: I 'member when y'all were writing this function yourselves throughout the compiler 😉

**Pull request merged**